src/pages/_app.tsx

```diff
@@ -6,6 +7,16 @@ import { theme, GlobalStyle } from '@/themes';
 import MainLayout from '@/components/MainLayout';
 
 function MyApp({ Component, pageProps }: AppProps) {
+  const [domLoaded, setDomLoaded] = useState(false);
+
+  useEffect(() => {
+    setDomLoaded(true);
+  }, []);
+
+  if (!domLoaded || typeof window === 'undefined') {
+    return null;
+  }
+
```

Comment:
hydration failed 에러가 발생해서 [관련링크](https://stackoverflow.com/questions/71706064/react-18-hydration-failed-because-the-initial-ui-does-not-match-what-was-render)보고 대응했습니다

Reply:
revert하기